src/conn/conn_api.c

```diff
@@ -2199,6 +2201,8 @@ wiredtiger_open(const char *home, WT_EVENT_HANDLER *event_handler,
 	__wt_scr_free(session, &i2);
 	__wt_scr_free(session, &i3);
 
+	if (free_merge_cfg)
+		__wt_free(session, merge_cfg);
```

Comment:
@sueloverso, if we initialize `merge_cfg` to NULL, I don't think we need the additional boolean (and we don't have to test before "calling" `__wt_free`, either.